### PR TITLE
npm script to clean all

### PR DESCRIPTION
**Pull request by gokullan**
This PR addresses issue #348. I have modified `package.json` to include the `clean:all` script which runs `rimraf` on all relevant cleaning-scripts. Do let me know if any changes are needed.

### CODE REVIEW

**Comment by kennarddh** [comment]
Can you use `concurrently` and call the other scripts instead of calling `rimraf`

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
 		"clean:logs": "rimraf ./Logs",
 		"clean:dist": "rimraf ./dist",
 		"clean:coverage": "rimraf ./coverage",
+		"clean:all": "rimraf ./build ./Logs ./dist ./coverage",
```
Comment (kennarddh) [inline package.json:16]: Can you use `concurrently` and call the other scripts instead of calling `rimraf` directly?

**Review by kennarddh — CHANGES_REQUESTED** [review]
Use concurrently and call the other scripts instead of calling rimraf

File: package.json (hunk 1 above)
Comment (gokullan) [reply package.json]: Sure, okay.

**Comment by gokullan** [comment]
Hi, I have made the requested changes. Do let me know if everything's fine.

**Comment by kennarddh** [comment]
```
$ npm run clean:all

> express-boilerplate@1.0.0 clean:all
> concurrently 'npm run clean' 'npm run clean:logs' 'npm run clean:dist' 'npm run clean:coverage'

[0] ''npm' is not recognized as an internal or external command,
[0] operable program or batch file.
[1] 'run' is not recognized as an internal or external command,
[1] operable program or batch file.
[2] 'clean'' is not recognized as an internal or external command,
[2] operable program or batch file.
[3] ''npm' is not recognized as an internal or external command,
[3] operable program or batch file.
[4] 'run' is not recognized as an internal or external command,
[4] operable program or batch file.
[5] The filename, directory name, or volume label syntax is incorrect.
[6] ''npm' is not recognized as an internal or external command,
[6] operable program or batch file.
[7] 'run' is not recognized as an internal or external command,
[7] operable program or batch file.
[8] The filename, directory name, or volume label syntax is incorrect.
[9] ''npm' is not recognized as an internal or external command,
[9] operable program or batch file.
[10] 'run' is not recognized as an internal or external command,
[10] operable program or batch file.
[11] The filename, directory name, or volume label syntax is incorrect.
[0] 'npm exited with code 1
[1] run exited with code 1
[2] clean' exited with code 1
[3] 'npm exited with code 1
[4] run exited with code 1
[5] clean:logs' exited with code 1
[6] 'npm exited with code 1
[7] run exited with code 1
[8] clean:dist' exited with code 1
[9] 'npm exited with code 1
[10] run exited with code 1
[11] clean:coverage' exited with code 1
```
-
I think you need to use double quote `"`

**Comment by gokullan** [comment]
Oh okay, it seemed to be working on my system. No issues, I'll change it right away.

**Comment by gokullan** [comment]
Oh okay. It worked on my local PC. No issues, I have changed it. This is the output I obtained presently.
```

> express-boilerplate@1.0.0 clean:all
> concurrently "npm run clean" "npm run clean:logs" "npm run clean:dist" "npm run clean:coverage"

[0] 
[0] > express-boilerplate@1.0.0 clean
[0] > rimraf ./build
[0] 
[1] 
[1] > express-boilerplate@1.0.0 clean:logs
[1] > rimraf ./Logs
[1] 
[2] 
[2] > express-boilerplate@1.0.0 clean:dist
[2] > rimraf ./dist
[2] 
[3] 
[3] > express-boilerplate@1.0.0 clean:coverage
[3] > rimraf ./coverage
[3] 
[0] npm run clean exited with code 0
[2] npm run clean:dist exited with code 0
[3] npm run clean:coverage exited with code 0
[1] npm run clean:logs exited with code 0
```

**Comment by kennarddh** [comment]
Thanks, it worked.

**Review by kennarddh — APPROVED** [review]

**Pull request merged**